src/init/features/dataconnect/index.ts

```diff
@@ -25,6 +25,12 @@ export async function doSetup(setup: Setup, config: Config): Promise<void> {
     type: "input",
     default: "dataconnect",
   });
+  // TODO: Guided prompts to set up connector auth mode and generate
+  const connectorId = await promptOnce({
+    message: "What ID would you like to use for your connector?",
+    type: "input",
+    default: "my-connector",
+  });
   // Hardcoded locations for when there is no project set up.
   let locationOptions = [
```

Comment:
We can move `locationOptions` block right before the prompt?

Don't need to list FDC locations when developer already picked the csql instance.

Reply:
Done.